packages/DropdownMenu/src/ContentContainer.styles.js

```diff
@@ -0,0 +1,23 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+
+const ContentContainerStyles = styled.div`
```

Comment:
The approach we've been leaning towards is to just export the raw styles, so this could simply be something like:
```js 
export default const contentStyles = css`
```
...but if you have an argument for why this approach is preferred, I'm all 👂👂

Reply:
I don't think you can do `export default const` you might still need to do the `const` but definitely, you can use `css''` instead